code/modules/power/engines/supermatter/supermatter.dm

```diff
@@ -554,10 +560,8 @@
 
 		if(prob(50))
 			var/mole_crunch_bonus = 0
-			if(combined_gas > MOLE_CRUNCH_THRESHOLD)
-				mole_crunch_bonus = 7000 //This adds 7000 EER worth of power to the SM. This should make mole crunch potentially worthy as a SM setup, if not risky. More stable than the anomalies, but very easy to push over the edge. Don't forget, a high EER setup can harvest power through zaps
-				if(!has_been_powered) // Nice try, no free no risk power
-					enable_for_the_first_time()
```

Comment:
Why remove this?

Reply:
This was added to prevent N2 crush from generating power without activation the engine. 
This PR changes the crush bonus to be proportional to EER, so it becomes redundant.

Reply:
I'm still hesitant on it, if its producing radiation it should be enabled so that admins can properly track logs.

Reply:
If EER is above 0 other checks are triggered. If EER is 0 no radiation is produced. The bonus is a multiple of EER, so with 0 EER the bonus is 0. Also, this checked for the gas amount being over the crunch threshold, but with the new behaviour the bonus grows proportionally with the amount of gas rather than cutting out completely once below the threshold.

Reply:
i still dont like it but i dont give enough of a shit